src/screens/FundCampaignPledge/PledgeColumns.tsx

```
@@ -0,0 +1,173 @@
+                />
+              )}
+              <span>{user.name}</span>
+            </div>
+          ))}
+          {extraUsers.length > 0 && (
+            <div
+              className={styles.moreContainer}
+              aria-describedby={id}
+              onClick={(event) => handleClick(event, extraUsers)}
+              data-testid={`moreContainer-${params.row.id}`}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)